### PR TITLE
SPCTR-3492 : Resolve style loading issue in fse 2024 theme header

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
Fix a style loading problem in the FSE 2024 theme header, This issue affects how the header looks and behaves, potentially affecting the theme's overall appearance and user experience.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-post-assets.php [hunk 1]
```diff
@@ -1453,4 +1460,30 @@ public function merge_array_string_values( $array1, $array2 ) {
...
+	 * @since X.X.X
+	 * @return array
+	 */
+	public function get_core_pattern_assets( $block ) {
+		if ( empty( $block['attrs']['slug'] ) ) {
+			return array();
+		}
+	
+		$slug     = $block['attrs']['slug'];
+		$registry = WP_Block_Patterns_Registry::get_instance();
```
[the remark this answers is not in the document]
Comment (rahulvarma722) [reply classes/class-uagb-post-assets.php:1483]: Checked

File: classes/class-uagb-post-assets.php [hunk 2]
```diff
@@ -1453,4 +1460,36 @@ public function merge_array_string_values( $array1, $array2 ) {
...
+		$slug = $block['attrs']['slug'];
+
+		// Check class and function exists.
+		if ( ! class_exists( 'WP_Block_Patterns_Registry' ) || ! method_exists( 'WP_Block_Patterns_Registry', 'get_instance' ) ) {
+			return array();
+		}
+		
+		$registry = WP_Block_Patterns_Registry::get_instance();
+	
+		if ( ! $registry->is_registered( $slug ) ) {
```
Comment (AbhijitNage123) [inline classes/class-uagb-post-assets.php:1485]: Need to add function/method exist for `is_registered`

File: classes/class-uagb-post-assets.php [hunk 3]
```diff
@@ -1453,4 +1460,36 @@ public function merge_array_string_values( $array1, $array2 ) {
...
+			return array();
+		}
+	
+		$pattern = $registry->get_registered( $slug );
```
Comment (AbhijitNage123) [inline classes/class-uagb-post-assets.php:1489]: Need to add function/method exist for `get_registered`

File: classes/class-uagb-post-assets.php (hunk 2 above)
Comment (rahulvarma722) [reply classes/class-uagb-post-assets.php:1485]: checked.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**